src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,38 @@
-export const UserInfo = () => {};
+import { User } from '../../types/types';
+
+type Props = {
+  users: User[];
+  setUser: (n: number) => void;
+  user: number;
+  errorUser: boolean;
+};
+
+export const UserInfo: React.FC<Props> = ({
+  users,
+  setUser,
+  user,
+  errorUser,
+}) => {
+  return (
+    <div className="field">
+      <select
+        data-cy="userSelect"
+        onChange={(e) => setUser(+e.target.value)}
```

Comment:
it is bad practise to make arrow function in jsx.
make handler in components and pass reference here
```
const handleChange = (e: React.ChangeEvent<HtmlSelectElement>) => setUser(+e.target.value);
```
```suggestion
        onChange={handleChange}
```